api/contract.go

```diff
@@ -16,6 +16,11 @@ const (
 	ContractStateFailed   = "failed"
 )
 
+const (
+	ContractUsabilityBad          = "bad"
+	ContractUsabilityGoodForRenew = "goodforrenew"
```

Comment:
I'd probably do `ContractUsabilityGood` which means "as good as it gets".
If we later decide that we need more like `GoodForRenew` we can add them but to me "GoodFor" always sounds like there is some restriction and that it's not just good for everything a contract can be good for.